posts/2024-12-10_road-to-mainnet/README.md

```diff
@@ -0,0 +1,344 @@
+
+- **Interoperability with IBC and ICS:** **gno.land** will integrate Inter-Blockchain
+Communication (IBC) and Interchain security (ICS), allowing seamless interaction 
+between **gno.land** and other IBC and ICS compatible chains. During this 
+milestone, modified versions of IBC and ICS will be integrated, versions that 
+are fully compatible with existing Cosmos chains, called IBCx and ICSx.
+- **Gno SDK:** Builders will gain access to the Gno SDK, making the entire process
+of building and deploying dApps on the network much easier.
+- **Launchpad for dApps:** The release of a specific Launchpad for dApps will 
+allow builders to quickly deploy and scale their applications on gno.land.
+- **Reward Distribution:** Contributors will be able to collect the rewards they
+have accumulated from different contributions until this phase of the network.
+- **Performance Enhancements:** The Mainnet Launch will also introduce significant
+performance upgrades, including sharding and increased TPS, to support more 
+complex applications.
+- **Transfers enabled:** During this phase of the launch, GNOT tokens become 
+transferable. This transition will mark a significant milestone, allowing users 
+to engage in token transactions, interact with the network and its applications, 
+and contribute to the broader economic activities within the gno.land ecosystem.
+- **Boards**: A full social interaction dApp will be available. Users will continue
+to create and participate in forums and message boards, with the added ability
+to build their own decentralized social media platforms using a customizable 
+framework. Forkability and DAO-driven governance will enable scalable, user-driven
+communities.
+
+## Call to Contribute
+
+gno.land puts at the center of its identity the contributors that are helping to
+create and shape the project. Contributing to gno.land’s development today means 
+you have the opportunity to make a notable impact at such a critical stage in the
+project’s roadmap.
+
+For now, we are looking for the earliest adopters; those curious to explore a 
+new way to build smart contracts and eager to be a part of the next generation of
+blockchain programming. Join us and [become a contributor today](https://gno.land/contribute)!
+
+## Conclusion
+
+With more than three years in the making, gno.land is now closer than ever to
+launching. The **Beta Launch,** while representing an important step in gno.land’s
+journey, should be seen as only the beginning.
+
+Future milestones are envisioned to bring improved performance, 
+interoperability with other blockchain networks through IBCx and ICSx, the 
+ability for builders to integrate their products and services with gno.land, 
+onboarding tools for different user segments, enhanced support for builders and 
+much more.
+
+We will host a Q&A on [Discord](https://discord.com/invite/gnoland), Friday, 
+December 13th at 5:30 pm CET, for the community to join and learn more about 
```

Comment:
Note: we are finding a new time for Tuesday to allow for promo time - this will need to be updated. cc @adrianakalpa